index.js

```diff
@@ -54,7 +79,26 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let averageIMDBRating = 0;
+  let counter = 0;
+  let ratings;
+
+  if (movies.length < 1) {
+    return averageIMDBRating;
+  } else {
+    for (let i = 0; i < movies.length; i++) {
+      for (let j = 0; j < movies[i].ratings.length; j++) {
+        ratings = parseFloat(movies[i].ratings[0].value);
```

Comment:
Why did we decide to iterate through `movies[I].ratings` if we only needed what was at index `[0]`?